### PR TITLE
feat: remove no longer needed commitmentHash from LightClientBridge c…

### PR DESCRIPTION
**Pull request by philipstanislaus**
…ontract

### CODE REVIEW

File: ethereum/contracts/LightClientBridge.sol [hunk 1]
```diff
@@ -291,10 +289,8 @@ contract LightClientBridge {
                 requiredNumOfSignatures
             );
 
-        /**
-         * @dev Encode and hash the commitment
-         */
```
Comment (philipstanislaus) [inline ethereum/contracts/LightClientBridge.sol:296]: This created an error, probably a bug in the Solidity compiler: 

```
./ethereum/contracts/LightClientBridge.sol:295:17: ParserError: Only state variables or file-level variables can have a docstring.
        bytes32 commitmentHash =
                ^------------^

Compilation failed. See above.
Truffle v5.3.8 (core: 5.3.8)
Node v14.16.1
```

Comment (musnit) [reply ethereum/contracts/LightClientBridge.sol:296]: weird

Comment (denalimarsh) [reply ethereum/contracts/LightClientBridge.sol:296]: Looks related to https://github.com/ethereum/solidity/issues/9139

**Comment by musnit** [comment]
deprecating - no longer relevant

File: ethereum/contracts/LightClientBridge.sol (hunk 1 above)
Comment (Vishwas1) [reply ethereum/contracts/LightClientBridge.sol:296]: same here coming with v0.8.0 how to solve this ?

**Pull request closed without merging**